openmls/src/schedule/mod.rs

```diff
@@ -208,6 +211,19 @@ impl From<Secret> for InitSecret {
     }
 }
 
+/// Creates a string from the given MLS `ProtocolVersion` for the computation of
+/// the `init_secret` when creating or processing a commit with an external init
+/// proposal.
+fn hpke_info_from_version(version: ProtocolVersion) -> String {
+    match version {
+        ProtocolVersion::Reserved => "Reserved",
+        ProtocolVersion::Mls10 => "MLS 1.0",
+        ProtocolVersion::Mls10Draft11 => "MLS 1.0 Draft 11",
```

Comment:
Please also flag the pre-release issue here, this conversion should happen elsewhere

Reply:
This conversion is specific to the derivation of the init secret. Why shouldn't it be here?

Reply:
Also, I'm not sure why the pre-release issue should be flagged here. This has to happen regardless of the pre-release issue.

Reply:
True, but the string prefix "MLS 1.0" is also used for hash-based identifiers for example. I just thought it would be easier to have it somewhere closer to `ProtocolVersion`. I also realise now the spec is not consistent in that regard, sometimes "mls10" is also used. I'll follow up with the spec.